Configurations/README

```diff
@@ -15,6 +15,30 @@ configuration in diverse ways:
                 information.
 
 
+Naming conventions for config files
+===================================
+
+Config files (*.conf) are named {nn}-{name}.conf, where {nn} is a two
+digit number and {name} is an indicator of what kind of configuration
+can be expected in the file.  The convention for {name} may be refined
+for some number groups.
```

Comment:
Perhaps add: "The numbers have no significance other than grouping. All config names across all files must be unique"

Reply:
Okie (or something along those lines at least), good idea

Reply:
Ah, I don't agree that names must be unique across all files.  For example, we have 10-linux.conf, which is the small set of linux configs we view as primary, but there is a number of them that we cannot claim, and I fully expect those to end up in, say, 50-linux.conf.  Why complicate matters further?

Reply:
Not the filenames, the configuration target names.

Reply:
Ahhhh, yeahok, I'll put the wording in